starters/blog/src/templates/blog-post.js

```diff
@@ -67,13 +67,21 @@ const BlogPostTemplate = ({ data, pageContext, location }) => {
 export default BlogPostTemplate
 
 export const pageQuery = graphql`
-  query BlogPostBySlug($slug: String!) {
+  fragment PrevNextFields on MarkdownRemark {
+    fields {
+      slug
+    }
+    frontmatter {
+      title
+    }
+  }
```

Comment:
Fragments in starters may be scary for new users?

Reply:
Good point I can just have 2 inline copies of those fields if that's preferred. @LekoArts what do you feel about that?

Reply:
Yeah, let's keep it simple instead of DRY :)

Reply:
Fixed in https://github.com/gatsbyjs/gatsby/pull/27359/commits/4025104f6f8ac0fedf356cf25a9798d5c0fb90c2